aip/general/0202.md

```diff
@@ -112,10 +145,29 @@ and see usage in Google APIs. Requiring such extra guidance means that governing
 the format specification is not the responsibility of the `FieldInfo.Format`
 enumeration itself.
 
+#### Why annotate type reference on generic-typed fields?
+
+Without this statically defined type information, tools consuming the API
+specification have no indicator of the types that the generic field could
+contain, limiting their ability to facilitate accurate API consumption. See
+[AIP-146 Declaring `Any` type references][generics] for more specific rationale.
+
+#### Why support a wildcard value for type references?
+
+Generic fields can be used to accept generic user input, in which case, the
```

Comment:
Again, I don't think it's only user input. I think it's find to still include this as an example, but maybe tweak to avoid implying that it would never be server-side.

Reply:
Great point, fixed!